wd_core.au3

```diff
@@ -1603,14 +1602,14 @@ Func __WD_Error($sWhere, $iErr, $sMessage = Default, $iExt = Default)
 			If $iErr <> $_WD_ERROR_Success Then ContinueCase
 
 		Case $_WD_DEBUG_Info
-			Local $sExtended = (($iExt) ? (" / " & $iExt) : (""))
+			Local $sExtended = (($iExt <> Default) ? (" / " & $iExt) : (""))
 			$sMsg = $sWhere & " ==> " & $aWD_ERROR_DESC[$iErr] & " ( " & $iErr & $sExtended & " )"
 			$sMsg &= (($sMessage) ? (" : " & $sMessage) : (""))
 			__WD_ConsoleWrite($sMsg)
 
 			If $iErr <> $_WD_ERROR_Success Then
 				If $_WD_ERROR_MSGBOX And $iErr < 6 Then
-					Local $iAnswer = MsgBox($MB_ICONERROR + $MB_OKCANCEL, "Webdriver UDF Error:", $sMsg)
+					Local $iAnswer = MsgBox($MB_ICONERROR + $MB_OKCANCEL, "au3WebDriver UDF Error:", $sMsg)

```

Comment:
au3WebDriver is the repo name. I still use WebDriver when referring to the UDF. 😉

Reply:
Take a look: [au3WebDriver > WebDriver](https://github.com/Danp2/au3WebDriver/pull/244/commits/5c3cb79b7b1a44e398daf36834502554d16813fc)